nx/source/services/nv.c

```diff
@@ -54,6 +54,9 @@ Result _nvInitialize(void) {
 
         if (R_SUCCEEDED(rc))
             rc = _nvCmdInitialize(CUR_PROCESS_HANDLE, g_nvTransfermem.handle, tmem_size);
+        
+        if (R_SUCCEEDED(rc))
```

Comment:
This has to be unconditional. Something like: `Result rc2 = tmemCloseHandle(...);` `if (R_SUCCEEDED(rc)) rc = rc2;`